src/styles/_base.scss

```diff
@@ -52,3 +52,11 @@ textarea:focus,
 button:focus {
   outline: none;
 }
+
+h3 {
+  font-weight: bold;
```

Comment:
And always use number for font weights as font families etc may not do what you expect with bold.